apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -231,7 +231,7 @@ export const NewHomeScreen = (): ReactElement => {
   const isInvalidUser = useUserForcedToLogout();
   useEffect(() => {
     if (!self) {
-      navigate("/new/login", { replace: true });
```

Comment:
make sure to have searched the entire codebase for the strings `new/` and `/new`